apps/api/src/passport/test/e2e.spec.ts

```diff
@@ -895,6 +899,60 @@ describe("E2E Tests", () => {
       });
     });
   });
+
+  describe("Login attempts", () => {
+    beforeEach(async () => {
+      const module = await Test.createTestingModule(moduleMetaData).compile();
+
+      req = module.get(Request);
+      app = module.createNestApplication();
+
+      await app.listen(req.socket);
+
+      // WAIT UNTIL IDENTITY IS INSERTED
+      await new Promise((resolve, _) => setTimeout(resolve, 3000));
+    });
+
+    const loginWithWrongPassword = () =>
+      req.post("/passport/identify", {
+        identifier: "spica",
+        password: "wrongPassword"
+      });
+
+    it("should block if the attempt limit is exceeded ", async () => {
+      const responses = [];
+      for (const fn of Array(3).fill(loginWithWrongPassword)) {
+        responses.push(await fn());
+      }
+
+      const responseWithBlockedError = responses.at(-1);
+
+      expect(responseWithBlockedError.statusCode).toEqual(401);
+      expect(responseWithBlockedError.statusText).toEqual("Unauthorized");
+      expect(responseWithBlockedError.body.message).toContain(
+        "Too many failed login attempts. Try again after"
```

Comment:
Could you use fake timers and update this expectation to see how many seconds are left?